Lib/test/test_uuid.py

```diff
@@ -514,14 +515,39 @@ class TestUUIDWithExtModule(BaseTestUUID, unittest.TestCase):
 class BaseTestInternals:
     uuid = None
 
+    @unittest.skipUnless(AIX, 'requires AIX')
+    def test_find_mac_netstat(self):
+        data = '''Name  Mtu   Network     Address           Ipkts Ierrs    Opkts Oerrs  Coll
+en0   1500  link#2      fe.ad.c.1.23.4   1714807956     0 711348489     0     0
+                        01:00:5e:00:00:01
+en0   1500  192.168.129 x071             1714807956     0 711348489     0     0
+                        224.0.0.1
+en0   1500  192.168.90  x071             1714807956     0 711348489     0     0
+                        224.0.0.1
+'''
+        popen = unittest.mock.MagicMock()
+        popen.stdout = io.BytesIO(data.encode())
+
+        with unittest.mock.patch.object(shutil, 'which',
+                                        return_value='/usr/bin/netstat'):
+            with unittest.mock.patch.object(subprocess, 'Popen',
+                                            return_value=popen):
+                mac = self.uuid._find_mac_netstat(
+                    command='netstat',
+                    args='-ia',
+                    hw_identifiers=b'Address',
+                    get_index=lambda x: x * 1,
```

Comment:
I don't understand why you need to use i+1 here. It looks like a bug.

Reply:
Guess my replies did not make it here.
a) It is "* 1" not plus 1
b) it may indicate a bug - when I was trying to use the find_mac code to parse netstat output I first tried "get_index=lambda x: x", but it always evaluated as get_index=lambda x: x + 1. I may have also tried x + 0, but I do not recall. Before merging I can experiment with other values - and if they are not working that is a different bug - not the one I am trying to resolve.